snippets/nrf70-fw-patch-ext-flash/fw-patch-ext-flash.overlay

```diff
@@ -3,16 +3,8 @@
  * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
  */
 
-&qspi {
-	mx25r64: mx25r6435f@0 {
-		partitions {
-			compatible = "fixed-partitions";
-			#address-cells = <1>;
-			#size-cells = <1>;
-			nrf70_fw_partition: partition@0 {
-				label = "nrf70_fw_partition";
-				reg = <0x0 DT_SIZE_K(128)>;
-			};
-		};
+/{
```

Comment:
```suggestion
/ {
```
Rather than re-run CI, please fix up in a future commit e.g. when sysbuild support is added to snippets